Resources/Audio/_Nuclear14/Weapons/Firing/attributions.yml

```diff
@@ -0,0 +1,4 @@
+﻿- files: ["war_never_changes.ogg"]
```

Comment:
This seems to be attributing a file that isn't added in this pr??

Reply:
Seems I forgot to change this.